### ISSUE TITLE
Running from a shared folder only works if ran as admin on latest 5 builds (17062-17066)

### ISSUE THREAD

**Issue by Spartan304** [RPCS3/rpcs3#16258]
### Quick summary

The latest 5 builds (17062-17066) will only work when ran from my shared drive if I right-click and run them as admin. 

### Details

 If I try to run them normally, the exe just shows up in task manager as a "Background process" and never proceeds to load the GUI.  If I run as admin, it loads as normal.  Build 17061 runs perfectly fine just by launching the exe.  The included log file is from a successful launch after running as admin as running it normally doesn't generate a log.

### Attach a log file

[RPCS3.log](https://github.com/user-attachments/files/17536955/RPCS3.log)

**Comment by Hasster1** [comment]
Additional info, since i have more or less the same experience;
The emulator only generates RPCS3.buf before being stuck as an empty background process that only takes about 500kb of ram, which prevents rpcs3 from being run again, giving the 'RPCS3 is already running', unless you delete that RPCS3.buf or end the task.
The bug also possibly occurs when using shortcuts, though i can't get any consistent results.
Also admin rights don't always help with it, for me it might just randomly work again after a while.

**Comment by Hasster1** [comment]
Also it doesn't have anything to do with a shared folder, moving rpcs3 into a newly created folder in the root of a C drive only lets you open it once (at least that's what happened to me), and then the same issue reappears.

**Comment by Monsterman07** [comment]
I sometimes get the bad builds running myself too. For me, I don't have any shared drives and all are local except if you count the terabox icon.

**Comment by Spartan304** [comment]
> I sometimes get the bad builds running myself too. For me, I don't have any shared drives and all are local except if you count the terabox icon.

I can try and run them 100 times and they will not load from my shared folder unless I run them as admin.

> Also it doesn't have anything to do with a shared folder, moving rpcs3 into a newly created folder in the root of a C drive only lets you open it once (at least that's what happened to me), and then the same issue reappears.

I don't have that issue at all on my local C drive.  I can run 17062-17065 every single time normally if it's on my local disk, only if I try to run them from shared.

**Comment by Spartan304** [comment]
Not sure if it's completely fixed or what exactly caused it, but I received an update to Windows Defender this evening and when I launched 17066 from C drive again, this time SmartScreen popped up.  I let it through and it loaded like normal.  I then copied the entire contents of the folder over to my shared drive, tried to launch it, and this time it worked.  So it looks like it's something about RPCS3 that Windows Defender/SmartScreen isn't happy about or when launching from a share it bugs out SmartScreen?  I manually updated 3 other apps on the same share and they all launched fine.
